app/models/reservation.rb

```diff
@@ -1,4 +1,16 @@
 class Reservation < ApplicationRecord
   belongs_to :user
   belongs_to :car
+
+  validates :date, :city, :duration, presence: true
+  validates :duration, numericality: { greater_than: 0 }
```

Comment:
[Optional] Since we are going to be working with a price per day for renting cars, it would be better if the duration also had a validation that makes sure its an integer. What do you think?

Reply:
Hi @Shisui6, yes I we need it to be an integer.